wpeverest/everest-forms
# Tweak - Enable minimum time for form submission

### All Submissions:





### Changes proposed in this Pull Request:

Previously, there was no feature to prevent the form submission if user submits the form before some duration. This PR add ability to prevent the form submission if there is no interaction of user on the form until the specified duration.

Closes # .

### How to test the changes in this Pull Request:

1. Go to Everest Forms Form Settings > Anti-Spam and Security.
2. Enable the **Minimum time for form submission**
3. Enter the duration(in seconds) for required interaction with the form.
4. Save the Form.
5. Check where it is restricting the form submission according to specified time or not in both ajax and non-ajax submission.

### Types of changes:




### Other information:




### Changelog entry

> Tweak - Enable minimum time for form submission
